src/components/PokeList.js

```diff
@@ -133,6 +140,7 @@ const PokeList = ({
 
   useEffect(() => {
     sortTypeCheck();
+    // eslint-disable-next-line
```

Comment:
you can also just disable eslint in your vs settings for now until end of project

Reply:
Please don't do that.  The linter is helpful except in this case.

Reply:
I know :). 👍